frontend/src/assets/css/typography.scss

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;600;700;800&display=swap");
```

Comment:
instead of using import, I recommend you
1.  add public/fonts folder, and add needed fonts inside in woff/woff2 format ( if you can't find in this format, there are a lot of converters online, use them )
2.  then add fonts.scss with @font-face rules inside
3. then reuse these rules inside styling h1-h6 etc